app/scripts/components/common/mapbox/map-options/basemaps.ts

```diff
@@ -15,8 +15,8 @@ export const BASEMAP_STYLES = [
   {
     id: 'satellite',
     label: 'Satellite',
-    mapboxId: 'cldu1cb8f00ds01p6gi583w1m',
-    thumbnailUrl: `https://api.mapbox.com/styles/v1/covid-nasa/cldac5c2c003k01oebmavw4q3/static/-9.14,38.7,10.5,0/480x320?access_token=${process.env.MAPBOX_TOKEN}`
+    mapboxId: 'ckb01h6f10bn81iqg98ne0i2y',
+    thumbnailUrl: `https://api.mapbox.com/styles/v1/covid-nasa/ckb01h6f10bn81iqg98ne0i2y/static/-9.14,38.7,10.5,0/480x320?access_token=${process.env.MAPBOX_TOKEN}`
```

Comment:
I'm not sure if this change is necessary to have things working locally but apparently I needed it to get things working. Should it be updated?